### ISSUE TITLE
Stone Gallagher does not take damage + Potential Quest Confusion

### ISSUE THREAD

**Issue by Quarris** [davqvist/Exoria#137]
When using In-World Crafting of the Stone Gallagher on Obsidian it did not take any damage at all (I mined 2 cobblestone previously). Did it on Exoria 0.4.1 on the server and managed to get nearly 3 stacks of obsidian chunks with still 260/262 durability left.

Also in the Quest "Strain your eyes" under "Survival" tab the quest mentions to get 'water' to use for strainers. I initially wasn't looking around for water but I looked around my base and I didn't see anything but Acid Water, so naively I tried to use it and it didn't work. Only after a while I decided to see if the world spawn natural water and then I managed to continue. It is probably just me being picky but maybe add something like "clean water" to the quest to make sure the player knows that they should find a pool of natural water and not acid one.

**Comment by davqvist** [comment]
The Hammer thing is a known issue of low priority,
I added "clean" to water in 0.4.2
